server/app/data.yml

```diff
@@ -7,6 +7,7 @@ defaults:
   config:
     title: Wiki
     host: http://localhost
+    listen_address: 0.0.0.0
```

Comment:
use camelCase as this is the standard for this project

Reply:
Fixed